protocol/x/prices/keeper/msg_server_create_oracle_market_test.go

```diff
@@ -28,7 +30,14 @@ func TestCreateOracleMarket(t *testing.T) {
 		expectedErr     string
 	}{
 		"Succeeds: create new oracle market (id = 1)": {
```

Comment:
nit: Add test case for market with a market pair that can't be converted to a currency pair?

Reply:
this test won't be relevant once i check that market exists in marketmap when trying to create oracle market for it (because i wouldnt able able to successfully create the market in marketmap as part of the setup function), so i'm opting not to include it here